tests/.env

```diff
@@ -4,8 +4,8 @@ TF_AWS_ACCESS_KEY=
 TF_AWS_REGION=
 TF_AWS_SECRET_KEY=
 TF_AZURE_APP_INSIGHTS_APP_ID=
```

Comment:
Why are we removing connectin string? We should use this for testing as well given instrumentation key is deprecated.

Reply:
Because it isn't used. I have just removed as not used. IDK if the SDK has support for it, I'll check it and update the tests if it has support, but in the other case I'd continue removing it